ui/src/components/UserIcon.js

```diff
@@ -19,38 +19,47 @@ import {LitElement, html, css} from 'lit-element';
 import {Icon} from '@material/mwc-icon';
 import {IconButton} from '@material/mwc-icon-button';
 
+import style from '../styles/components/UserIcon.css';
+
 /**
- * Button responsible for enabling the user to record and upload audio files. 
+ * Button responsible for enabling the user to record and upload audio files.
  */
 export class UserIcon extends LitElement {
+    static get properties() {
+        return {
+            userId: {type: Number},
+        };
+    }
 
     static get styles() {
-        return css`
-            mwc-icon-button {
-                --mdc-icon-button-size: 60px;
-                --mdc-icon-size: 30px; 
-                color: #3c4043;
-            }
-        `;
+        return style;
     }
 
     /**
      * Emits an event that causes the record page to close and the user
-     * form to appear. 
+     * form to appear.
      */
     handleUserIconClick() {
-        const event = new CustomEvent('enter-form', {});
+        const event = new CustomEvent('enter-form', {
+            bubbles: true,
+            composed: true,
+        });
         this.dispatchEvent(event);
     }
 
     render() {
         return html`
-            <mwc-icon-button 
-                icon="account_circle"
-                @click=${this.handleUserIconClick}
-            </button>
+            <div @click=${this.handleUserIconClick}>
+                <mwc-icon-button icon="account_circle"> </mwc-icon-button>
+
+                <span>
```

Comment:
I believe this is only for the non-mobile version--maybe make just the icon button the default?

Reply:
it's `display: hidden` by default